lua/astrocommunity/pack/eslint/README.md

```diff
@@ -0,0 +1,14 @@
+# ESLint
+
+[ESLint](https://eslint.org/) is a tool for identifying and reporting on
+patterns found in ECMAScript/JavaScript code, with the goal of making code more
+consistent and avoiding bugs.
+
+ESLint is completely pluggable. Every single rule is a plugin and you can add
+more at runtime. You can also add community plugins, configurations, and
+parsers to extend the functionality of ESLint.
+
+## Prerequisites
+
+To use ESLint, you must have [Node.js](https://nodejs.org/en/) (^18.18.0, ^20.9.0, or >=21.1.0) installed
+and built with SSL support. (If you are using an official Node.js distribution, SSL is always built in.)
```

Comment:
I believe this might be wrong per the https://github.com/microsoft/vscode-eslint docs. Version 3 appears, if I read the docs right, to require nodejs 20? 
https://github.com/microsoft/vscode-eslint#version-3010---release